sdk/spring/spring-cloud-azure-starter-active-directory-b2c/pom.xml

```diff
@@ -141,4 +141,24 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>3.1.2</version> <!-- {x-version-update;org.apache.maven.plugins:maven-jar-plugin;external_dependency} -->
+        <configuration>
+          <archive>
```

Comment:
Should we keep the same usage with the SDK parent configuration? SDK uses another attribute `<archive combine.children="append">`.

Reply:
Perhaps we want some children to use a common resource directory and also define additional ones. For this, we can append the parent configuration by using `combine.children=append` on the resources element in the parent. You can control how child POMs inherit configuration from parent POMs by adding attributes to the children of the configuration element. But `Spring Cloud Azure Starters` will not be used as a parent POM and there is no parent POM now. This is [documented in the pom reference](https://maven.apache.org/pom.html#Plugins).

Reply:
I don't think we need this, the sdk parent pom is a parent pom and expcts other pom to extend it. The same situation will never happens to our starter's poms.